pkg/network/server.go

```diff
@@ -1252,7 +1252,7 @@ func (s *Server) requestBlocks(bq Blockqueuer, p Peer) error {
 	lq := s.bQueue.lastQueued()
 	if lq > pl.IndexStart {
```

Comment:
`>=` here? We already have `pl.IndexStart` if they're equal and we should be requesting the next one.

Reply:
Fixed.